third_party/opa/data-and-policies/wkcpolicy.rego

```diff
@@ -0,0 +1,75 @@
+package wkcpolicy
```

Comment:
We shouldn't have anything wkc specific in the open source.  I'm guessing this is just a naming issue